client/src/components/footer/Footer.jsx

```diff
@@ -4,31 +4,42 @@ import logo_HYF from '../../logo_HYF.svg';
 import logoVox from '../../assets/logoVox.png'
 import { BsEnvelope } from 'react-icons/bs';
 import { FaGithub } from 'react-icons/fa';
+import './Footer.css';
 
 function Footer() {
 
 
     const [state, setState] = useState({
         email: "mailto:voxbox@gmail.com",
         repoUrl: "https://github.com/gelilaa/VoxBox",
+        homePage: 'https://voxbox.herokuapp.com'
      });
 
         return ( 
 
-            <div style = {{backgroundColor:'#397e6b', width: '100%', marginTop: 'auto'}} className = "footer text-white row p-2 text-center text-md-left" >
-            <div className = "col-md-10" style={{paddingLeft:"50px"}}>
-            <ul style={{listStyleType: 'none'}}>
-            <li className='mb-4 font-weight-bold'>Contact VoxBox</li>
-            <li><FaGithub /><a href = { state.repoUrl } className = "text-white" > https://github.com/gelilaa/VoxBox </a></li>
-            <li><BsEnvelope /><a href = { state.email } className = "text-white" > info.voxbox@gmail.com </a></li>
+            <div className = "footer-outer row p-2 text-center text-md-left" style={{backgroundColor: '#397e6b'}}>
+            
+            <div className = "col-4 " style={{backgroundColor: '#397e6b'}}>
+            <ul className='footer-list' style={{paddingTop:'20px'}}>
+            <li ><a href = { state.homePage } >VoxBox</a></li>
+            <li><a href = { state.repoUrl } > <FaGithub /> </a></li>
+            <li><a href = { state.email } > <BsEnvelope /> </a></li>
             </ul>
             </div> 
 
-            <div className = "col-md-2 m-auto-0 me-n1">
-            <img src = {logoVox} style = {{height:'4em', width:'4em'}} 
-            alt = "Voxbox logo" className='m-4 rounded'/>
-            <img src = {logo_HYF} style = {{height:'5em', width:'5em'}} 
-            alt = "HYF logo" /></div>
+            <div className = "col-4 " style={{backgroundColor: '#397e6b'}}>
```

Comment:
This Copyright div is not exactly in the center on my machine, so I tried with __text-center__ next to the col-4 and it was better. Maybe you can check this.

Reply:
> This Copyright div is not exactly in the center on my machine, so I tried with **text-center** next to the col-4 and it was better. Maybe you can check this.

Thank you! Fixing it now..